### ISSUE TITLE
跑的差距与论文中差距大

### ISSUE THREAD

**Issue by kuyedie520** [chenhao2345/ABMT#1]
为什么我用同样的ibn_a去按照你的代码跑的测试结果duke-market只有74.8%，差距有点大

**Comment by kuyedie520** [comment]
跑了很多次都跑不到你那个结果，想问一下你中间加了什么策略吗？还有你的imagenet预训练权重的加载是在resnet_ibn_AB网络里面就加载了吗？那你的源域训练代码中if args.resume:checkpoint = load_checkpoint(args.resume)  #这里是什么作用？

**Comment by chenhao2345** [comment]
你好， 我这边重新跑了下，结果和我文章里差不多。我刚刚更新了下readme，请参考Example2：
#### Stage I: Pre-training on the source domain

```shell
sh ABMT_source_pretrain.sh dukemtmc-reid market1501 resnet_ibn50a_AB
```

#### Stage II: End-to-end training with ABMT 

```shell
sh ABMT_target_adaptation.sh dukemtmc-reid market1501 resnet_ibn50a_AB
```
我并没有用什么别的技巧。我这边是pytorch1.2， 4块GPU。resnet_ibn_AB里面就加载了下载的权重，跟源域训练里if args.resume:checkpoint = load_checkpoint(args.resume)无关。

**Comment by kuyedie520** [comment]
那可以问一下在源域训练环节中if args.resume:checkpoint = load_checkpoint(args.resume)是什么意思吗？非常感谢你的回复

**Comment by chenhao2345** [comment]
源域训练环节中if args.resume:checkpoint = load_checkpoint(args.resume)只是留着测试训练好的模型用的，你可以用它来读入一个训练好的模型来继续训练或者测试。跟文章内容并没有关系。

**Comment by kuyedie520** [comment]
非常感谢你，我开始是在两块GPU上跑的差距很大，现在换成四块GPU，性能提升很大，与论文中差不多，再次感谢您的回复。

**Comment by chenhao2345** [comment]
不用客气，batch size的大小和GPU的数量会影响batch normalization的功效。

**Comment by kuyedie520** [comment]
你好，根据你的经验batch size最大设置为多少效果比较好，128？

**Comment by chenhao2345** [comment]
貌似一个GPU上训练16张图比较好，也就是说2个GPU32张，4个GPU64张，我是从这篇文章里面了解到的
[MMT (ICLR 2020)](https://github.com/yxgeee/MMT)

**Comment by kuyedie520** [comment]
非常感谢你，抱拳了
